File: 8958 OX퀴즈/8958.py
n = int(input())
for i in range(0, n):
    inputStr = str(input())
    accum = 0
    answer = 0
    for c in inputStr:
        if c == 'O':
            accum += 1
            answer += accum
        else:
            accum = 0
    print(answer)

File: 8958 OX퀴즈/8958.java
import java.io.*;
import java.util.Scanner;

public class Main {
    static public void main(String[] args) throws IOException {
        Scanner scanner = new Scanner(System.in);
        int n = scanner.nextInt();

        for (int i = 0; i < n; ++i) {
            String input = scanner.next();
            int inputLength = input.length();
            int accum = 0;
            int answer = 0;
            for (int j = 0; j < inputLength; ++j) {
                if (input.charAt(j) == 'O') {
                    ++accum;
                    answer += accum;
                } else {
                    accum = 0;
                }
            }

            System.out.println(answer);
        }
    }
}
